app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_one :role
```

Comment:
Can a user have no role? 
If every user needs a role than you will also need to add a validation for that